sgx/seal_test.go

```diff
@@ -0,0 +1,28 @@
+package sgx_test
+
+import (
+	"github.com/medibloc/panacea-doracle/sgx"
+	"github.com/stretchr/testify/require"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestSealAndUnsealFile(t *testing.T) {
+	data := []byte("hello world")
+
+	testDir := "/test"
```

Comment:
I used a temp dir, "/test", for testing.
Since `ego sign <executable>`, not signing using `enclave.json`, the default `enclave.json` is created.
So `$HOME` env var is not set in test environment and I can't use home dir.
If you have any better idea, please let me know 🙏

Reply:
It's okay to use the `/test` because that directory doesn't need to be permanently maintained after this test is finished. 

In general, you can create a temporary file with a random filename. In Linux, temporary files are created in the `/tmp` (aka. tmpfs). 
https://gist.github.com/youngjoon-lee/c1d79ddc83e993b3fba91631b034083a

(I'm not sure if we can use `/tmp`, if we don't mount memfs to `/tmp` using `enclave.json`.)

Reply:
Right. I first tried to use `os.MkdirTemp` which makes directory under `/tmp`.
But I got an error of `stat /tmp: no such file or directory`. Thus I should have created directory `/tmp` as well.
It seems that there is no directory of `/tmp` in enclave.
If it would be better to change the directory name to `/tmp` instead of `/test`, please let me know 😄

Reply:
According to this [link](https://docs.edgeless.systems/ego/#/reference/config?id=mounts), the default enclave.json uses `memfs` type and only empty `/` exists.

Reply:
Ja. Then, keep your code as it is, because this test won't work on non-SGX environments anyway.